contrib/python-werkzeug/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-werkzeug"
+pkgver = "2.3.8"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-flit_core",
+]
+pkgdesc = "WSGI swiss-army knife"
+maintainer = "firefly-cpp <iztok@iztok.space>"
+license = "MIT"
+url = "https://werkzeug.palletsprojects.com"
+source = f"$(PYPI_SITE)/w/werkzeug/werkzeug-{pkgver}.tar.gz"
+sha256 = "554b257c74bbeb7a0d254160a4f8ffe185243f52a52035060b761ca62d977f03"
+# missing dependencies (pytest-timeout)
```

Comment:
package that then

Reply:
This is a plan. However, it is a pretty long chain to cover all dependencies of dependencies.

Reply:
if it's only pytest-timeout then the chain ends there (unless you want to run the pytest-timeout tests too, then it's pexpect+ptyprocess as well)

Reply:
Let's package pytest-timeout without running its tests for now. I am on it.

Reply:
Tests are now enabled